storybook/stories/components/search/nav.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+import React from 'react'
+import { Col, Layout, Grid, Root } from 'reflex'
+import styles from '../../../shared/stories.css'
+
+export default function SearchNav() {
+  return (
+    <Layout className={styles.colors2} margin={[1, 0, 0, 0]}>
```

Comment:
Actually just seeing this made me wonder, do we need margin / padding for layout? Can we implement the current designs without it at all?